.travis.yml

```diff
@@ -0,0 +1,56 @@
+# Used old infrastructure, needed for integration tests:
+# http://docs.travis-ci.com/user/workers/standard-infrastructure/
+sudo: required
+language: python
+python:
+  - 2.7
+
+branches:
+  only:
+    - master
+    - /^v[0-9]+\.[0-9]+$/
+
+env:
+  global:
+    - CACHE_NAME=JOB1
+  matrix:
+    - TASK=ci-unit NODE_INDEX=0 NODE_TOTAL=2
+    - TASK=ci-unit NODE_INDEX=1 NODE_TOTAL=2
```

Comment:
Did you also try splitting in in more than two chunks?

But I guess right now the lower time limit is the time it takes to run integration tests which we can't split into chunks that easily.